client/src/components/Messages.jsx

```
@@ -0,0 +1,27 @@
+const Messages = () => {
+    useGetMessages();
+    useGetRealTimeMessage();
+    const { messages } = useSelector(store => store.message);
+    return (
+        <div className='px-4 flex-1 overflow-auto'>
+            {
+               messages && messages?.map((message) => {
+                    return (
+                        <Message key={message._id} message={message} />
+                    )
+                })
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)